dpti/__init__.py

```
@@ -12,4 +12,5 @@
     "ti",
     "ti_water",
     "lib",
+    "workflows",
 ]
```

Ruff (0.11.9)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)